scripts/check-health.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# assert on the health of a running container
+IS_SYNCING=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.isSyncing')
+SHOULD_HAVE_PEERS=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.shouldHavePeers')
+PEERS=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.peers')
+
+if [ "$IS_SYNCING" != "true" ]; then
+    echo "FAIL"
+    exit 1
+fi
+
+if [ "$SHOULD_HAVE_PEERS" != "true" ]; then
+    echo "FAIL"
+    exit 2
+fi
+
+if [[ ! "$PEERS" -gt 0 ]]; then
```

Comment:
Maybe you could gate this check so as not to break running a local dev chain (with no expected peers). Something like only check peers > 0 if shouldHavePeers is true.

Reply:
Hmm, the intent here was to run this check in CI while the docker container is only pointed to mainnet as a sanity check that we didn't break the docker image. 

How about a BIG FAT disclaimer at the top of the file documenting what the intended purpose is ? Is that acceptable ?